tests/docs/test_supported_confs.py

```diff
@@ -308,8 +309,6 @@ def test_none_all(self, diff_method, return_type, shots, wire_specs):
                     msg = "not accepted with finite shots"
                 else:
                     msg = "Finite shots are not supported with"
-            elif return_type not in ("Hermitian", "Projector", Expectation):
-                msg = "not accepted for analytic simulation"
```

Comment:
Are these now supported?

Reply:
Those were the cases supported before. Now we support either:
1. Any combination of expectation values of things that have a matrix
2. Any combination of measurements that are all in the Z measurement basis